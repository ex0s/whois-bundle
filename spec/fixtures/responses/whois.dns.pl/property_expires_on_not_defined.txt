
DOMAIN NAME:           nom.pl
registrant type:       organization
nameservers:           f-dns.pl. [217.17.46.189][2001:1a68:0:10::189]
                       i-dns.pl. [156.154.100.15]
                       a-dns.pl. [195.187.245.44]
                       e-dns.pl. [195.80.237.162]
                       d-dns.pl. [213.172.174.70]
                       h-dns.pl. [2001:678:4::2][194.0.1.2]
                       g-dns.pl. [2001:6d8:0:1::a:6][149.156.1.6]
created:               2003.03.25 12:00:00
last modified:         2004.02.09 12:24:22
renewal date:          not defined

no option

dnssec:                Signed


REGISTRAR:
NASK
ul. Wawozowa 18
02-796 Warszawa
Polska/Poland
+48.22 3808300
info@dns.pl

WHOIS displays data with a delay not exceeding 15 minutes in relation to the .pl Registry system
Registrant data available at http://dns.pl/cgi-bin/en_whois.pl