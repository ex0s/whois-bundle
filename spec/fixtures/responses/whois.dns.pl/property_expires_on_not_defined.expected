#expires_on
  %s == nil
